man/sd_bus_message_read_strv.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="sd_bus_message_read_strv">
+
+  <refentryinfo>
+    <title>sd_bus_message_read_strv</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_bus_message_read_strv</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_bus_message_read_strv</refname>
+
+    <refpurpose>Access an array of strings in a message</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-bus.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int <function>sd_bus_message_read_strv</function></funcdef>
+        <paramdef>sd_bus_message *<parameter>m</parameter></paramdef>
+        <paramdef>char ***<parameter>l</parameter></paramdef>
+      </funcprototype>
+    </funcsynopsis>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para><function>sd_bus_message_read_strv()</function> gives access to an array of strings in message
+    <parameter>m</parameter>. The "read pointer" in the message must be right before an array of strings. A
+    pointer to the <constant>NUL</constant>-terminated array of strings is returned in the output parameter
+    <parameter>l</parameter>. Note that ownership of any string array stored in <parameter>l</parameter> by
+    <function>sd_bus_message_read_strv()</function> is transferred to the caller. Hence, the caller is
+    responsible for freeing the results returned by this function.</para>
```

Comment:
I think this should be clarified whether the array itself, or the array including the strings must be freed by the caller. (It's the latter.)